src/app/cpr/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ if(NOT DRISHTI_BUILD_MIN_SIZE)
     target_compile_definitions(train_cpr PUBLIC DRISHTI_USE_IMSHOW)
   endif()
   
-  set_property(TARGET train_cpr PROPERTY FOLDER "app/console")
+  set_target_properties(train_cpr PROPERTIES FOLDER "app/console" INTERPROCEDURAL_OPTIMIZATION DRISHTI_EXE_IPO)
```

Comment:
From documentation:
* https://gcc.gnu.org/onlinedocs/gcc-5.4.0/gcc/Optimize-Options.html#Optimize-Options

```
Note that when -fno-fat-lto-objects is enabled the compile-stage is faster
but you cannot perform a regular, non-LTO link on them
```

So we can try to remove `-fno-fat-lto-objects` so we can link with regular executables but it will slow down our compile stage (?)